core/sql/sqlcomp/CmpSeabaseDDLcommon.cpp

```diff
@@ -3422,7 +3440,7 @@ Int64 CmpSeabaseDDL::getObjectUIDandOwners(
     strcpy(cfvd, " and valid_def = 'Y' ");
 
   char buf[4000];
```

Comment:
All the SQL requests generated by the DDL component use this method to create strings on the fly.  It is a good suggestion but we may want to consider some generic method and change all the string creations to use this method.  Do we really have this much of a concern for stack overflow?